src/train.py

```diff
@@ -113,7 +113,15 @@ def main(args):
 
     # Set logger
     _logger = wandb if not args.experiment.debug else Mock()
-    _logger.init(project="finding_willy", name=args.experiment.name)
+    if args.experiment.name:
```

Comment:
Maybe wrap the logging handling into function under utils? Too much code to be on main, imo
btw, do you think we should have utils folder and seperate the functions to different files with relavant grouping?

Reply:
Think that stuff about logging metrics should move to it's own place